src/vm/method.cpp

```diff
@@ -4005,18 +3997,6 @@ void MethodDesc::CheckRestore(ClassLoadLevel level)
 
             g_IBCLogger.LogMethodDescWriteAccess(this);
 
-            // If this function had already been requested for rejit, then give the rejit
-            // manager a chance to jump-stamp the code we are restoring. This ensures the
-            // first thread entering the function will jump to the prestub and trigger the
-            // rejit. Note that the PublishMethodHolder may take a lock to avoid a rejit race.
-            // See code:ReJitManager::PublishMethodHolder::PublishMethodHolder#PublishCode
-            // for details on the race.
-            // 
-            {
-                PublishMethodHolder publishWorker(this, GetNativeCode());
-                pIMD->m_wFlags2 = pIMD->m_wFlags2 & ~InstantiatedMethodDesc::Unrestored;
```

Comment:
Doesn't the flag update still need to happen?

Reply:
Yeah, it seems to